torch/nn/init.py

```diff
@@ -5,11 +5,47 @@
 from torch.autograd import Variable
 
 
+def calculate_gain(nonlinearity, param=None):
+    """Return the recommended gain value for the given nonlinearity function. The values are as follows:
+    ============ =========================================
+    nonlinearity gain
```

Comment:
I'd suggest to add `nonlinearity == "linear"` (or `None`) for conv layers without activation